load.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Workflow Module.
+ *
+ * @package hm-platform/workflow
+ */
+
+namespace HM\Platform\Workflow;
+
+use function HM\Platform\register_module;
+
+require_once __DIR__ . '/inc/namespace.php';
+
+// Do not initialise if plugin.php hasn't been included yet.
+if ( ! function_exists( 'add_action' ) ) {
+	return;
+}
+
+add_action( 'hm-platform.modules.init', function () {
+	$default_settings = [
+		'enabled' => false,
```

Comment:
I'm wondering if it might be better to enable this by default, but just not add the default notifications? Just wondering for general use cases, this module seems pretty good, as it's just additive, it remove anything, and it's there if people want to set up notifications etc.

Reply:
Yeah I guess so. We should rethink the module tiers RFC a bit in that case I think.